Tests/scripts/check_jobs_done.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+from argparse import ArgumentParser
+from os import getenv
+import logging
+
+
+NIGHTLY_JOBS = [
```

Comment:
Can all these be in config files?
That way, you won't need to touch a .py file.